.circleci/config.yml

```diff
@@ -15,6 +15,7 @@ job-build: &job-build
       - checkout
       - run: sudo -E apt-get update && sudo -E apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libpng-dev && sudo -E docker-php-ext-install -j$(nproc) iconv && sudo -E docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && sudo -E docker-php-ext-install -j$(nproc) gd
       - run: .circleci/build.sh
+      - run: .circleci/lint.sh
```

Comment:
lint is split into another script to allow running local linting without running tests and the other way around